modules/c1xBidAdapter.js

```diff
@@ -48,15 +49,30 @@ export const c1xAdapter = {
       response: 'json',
       compress: 'gzip'
     }
-    Object.assign(payload, tagObj);
 
-    let payloadString = stringifyPayload(payload);
+    if (pixelId) {
```

Comment:
Why is there sync added here, when the sync is also added on line 59?  Adding two syncs seems wasteful.